utils/vscode/snippets/carbon.json

```
@@ -0,0 +1,89 @@
+{
+    "Create a Function": {
+        "prefix": "function",
+        "body": [
+            "fn ${1:name} {",
+            "   return $2;",
```

please use $0 placeholder for the bodies. (same for other items)
```suggestion
            "   $0",
```